tests/vendor_tests/redhat/foreman_tests.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+from sos_tests import StageOneReportTest, redhat_only
+
+
+class ForemanBasicTest(StageOneReportTest):
+    """Ensure that a basic execution runs as expected with all TFM related
+    plugins
+
+    :avocado: tags=foreman
+    """
+
+    sos_cmd = '-v'
+
+    def test_tfm_plugins_ran(self):
+        self.assertPluginIncluded([
```

Comment:
Should it also assert the Apache and PostgreSQL plugins are loaded?

Reply:
+1 to both apache and postgresql plugins.

Reply:
Ack